### PR TITLE
WEB-1459: Fix course links 404 if the user is not yet logged in

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** 
An enrolled student is getting Page not Found in Labster.com when the user opened the "You have been enrolled in ..." email and click the course link.
It only happens if user is NOT logged in yet.

**JIRA:** https://liv-it.atlassian.net/browse/WEB-1459

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**

1. Open page A
2. Do thing B
3. Expect C to happen
4. If D happened instead - check failed.

**Reviewers:**
- [ ] tag reviewer
- [ ] tag reviewer

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.

### CODE REVIEW

**Comment by madyasiwi** [comment]
Have you tested this?

File: lms/djangoapps/courseware/views/views.py [hunk 1]
```diff
@@ -261,6 +261,7 @@ def jump_to(_request, course_id, location):
     return redirect(redirect_url)
 
 
+@login_required
```
Comment (TamoshaytisV) [inline lms/djangoapps/courseware/views/views.py:264]: What user will see in this case?

**Comment by lukyfebrianto** [comment]
> Have you tested this?

@madyasiwi  Yes, I test it locally and it works fine.
@TamoshaytisV if user not yet logged in it'll redirect to the login page with `next` parameter
see,
screenshot from 2018-08-30 10-48-31

**Review by madyasiwi — APPROVED** [review]
> @madyasiwi Yes, I test it locally and it works fine.

Ok, I am too lazy to setup edx locally to test it myself.

File: lms/djangoapps/courseware/views/views.py (hunk 1 above)
Comment (lukyfebrianto) [reply lms/djangoapps/courseware/views/views.py:264]: @TamoshaytisV answered, please see my comment below. Thanks

**Review by TamoshaytisV — APPROVED** [review]

**Pull request merged**